htdocs/wp-content/themes/owc-formulieren/assets/js/editor/news/edit.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useBlockProps, RichText } from '@wordpress/block-editor';
+import { __experimentalDivider as Divider, __experimentalNumberControl as NumberControl } from '@wordpress/components';
+
+const Edit = ( props ) => {
+	const { attributes, setAttributes } = props;
+	const { apiUrl, amountOfPosts, detailPageUrl } = attributes;
+
+	const blockProps = useBlockProps( {
+		className: 'openpdd-news',
+		style: {
+			backgroundColor: '#fff',
+			padding: '1.5rem',
+			border: '1px solid rgba(0, 0, 0, 0.1)',
+		},
+	} );
+
+	const isValidUrl = urlString => {
+		const urlPattern = new RegExp( '^(https?:\\/\\/)?' + // validate protocol
+		                               '((([a-z\\d]([a-z\\d-]*[a-z\\d])*)\\.)+[a-z]{2,}|' + // validate domain name
+		                               '((\\d{1,3}\\.){3}\\d{1,3}))' + // validate OR ip (v4) address
+		                               '(\\:\\d+)?(\\/[-a-z\\d%_.~+]*)*' + // validate port and path
+		                               '(\\?[;&a-z\\d%_.~+=-]*)?' + // validate query string
+		                               '(\\#[-a-z\\d_]*)?$', 'i' ); // validate fragment locator
+		return !!urlPattern.test( urlString );
+	};
+
+	const onApiUrlChange = ( url ) => {
+		if ( !isValidUrl( url ) ) {
+			return;
+		}
+
+		setAttributes( {
+			apiUrl: url,
+		} );
+	};
+
+	const onDetailPageUrl = ( url ) => {
+		if ( !isValidUrl( url ) ) {
+			return;
+		}
+
+		setAttributes( {
+			detailPageUrl: url,
+		} );
+	};
+
+	const onAmountOfPostsChange = ( amount ) => {
+		setAttributes( {
+			amountOfPosts: parseInt( amount ),
+		} );
+	};
+
+	return (
+		<div {...blockProps}>
+			<h1>
+				{__( 'Persoonlijk nieuws', 'owc-formulieren' )}
+			</h1>
+
+			<Divider />
+
+			<label>
+				<strong>
+					{__( 'API Url', 'owc-formulieren' )}
+				</strong>
+			</label>
+			<RichText
```

Comment:
`<TextControl />` met een `label` prop is mooier. 

https://wp-gb.com/textcontrol/